TLM/TLM/Util/AutoTimedTrafficLights.cs

```diff
@@ -0,0 +1,410 @@
+namespace TrafficManager.Util {
```

Comment:
Entire logic in this module deserves a couple paragraphs in our documentation, explaining the auto TL feature to the player how the traffic light is defined. From the player perspective. With a couple of examples. Ask @aubergine10 where to place this explanation.